source/features/faster-pr-diff-options.tsx

```diff
@@ -1,13 +1,17 @@
 import React from 'dom-chef';
 import select from 'select-dom';
 import * as pageDetect from 'github-url-detection';
-import {BookIcon, CheckIcon, DiffIcon} from '@primer/octicons-react';
+import {BookIcon, CheckIcon, DiffIcon, DiffModifiedIcon} from '@primer/octicons-react';
 
 import features from '.';
 
+function isPRPage(): boolean {
+	return !pageDetect.isSingleCommit() && !pageDetect.isCompare();
+}
+
 function createDiffStyleToggle(): DocumentFragment {
 	const url = new URL(location.href);
-	const isUnified = select.exists([
+	const isUnified = url.searchParams.get('diff') === 'unified' || select.exists([
```

Comment:
This seems redundant. Does it work without this check? The next line already has a selector for "form in PR"

Reply:
This fixed a bug I noticed while working on this PR:

> Also, noticed this bug where the wrong diff style button is selected when "No whitespace" is enabled (but only on compare and commit pages), e.g. : https://github.com/sindresorhus/refined-github/compare/21.4.13...21.4.23?w=1.

I'll look into it to see if this can be simplified.

Reply:
The second selector needed fixing (the `diff` parameter in the URL doesn't necessarily occurs at the end).